docs/en/collector/http-client.md

```diff
@@ -0,0 +1,60 @@
+# Collectors
+
+## [HttpClient collector](./../../../src/Collector/HttpClientCollector.php)
```

Comment:
Since it is a doc about a single collector, such title is better:

```suggestion
# [HttpClient collector](./../../../src/Collector/HttpClientCollector.php)
```